cdts/pam-devel-alma8-x86_64/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: pam-devel-alma8-x86_64
```

Comment:
Let's name this with `conda` instead of `alma8` as discussed in core meeting.

Reply:
Done in https://github.com/conda-forge/cdt-builds/pull/70/commits/b42c06110e18016d294e37e0fe8ba641fa281fda ff.